app/lib/bk/compat/parsers/bitbucket.rb

```diff
@@ -67,6 +54,14 @@ def parse
 
       private
 
+      def register_translators
```

Comment:
due to ruby's conventions, methods that modify an objects state should end with `!`:
```suggestion
      def register_translators!
```
Note that this should be applied to all parsers

Reply:
Applied 👍